requirements.txt

```diff
@@ -4,4 +4,6 @@ git+https://github.com/flesser/django-simple-history@django2.0
 psycopg2==2.7.3
 pytz==2017.3
 djangorestframework==3.7.7
+fuzzywuzzy
+python-Levenshtein

```

Comment:
I just looked up the documentation, it doesn't seem like we need this package? But fuzzywuzzy will install it automatically with `fuzzywuzzy[speedup]`

Reply:
yes, it is not necessary but it will be faster :)